containers/ecr-viewer/src/app/view-data/components/EcrDocument/clinical-data.tsx

```diff
@@ -238,27 +247,43 @@ export const returnCareTeamTable = (
     { columnName: "Dates", infoPath: "careTeamParticipantPeriod" },
   ];
 
-  careTeamParticipants.forEach((entry) => {
-    if (entry?.period) {
-      const { start, end } = entry.period;
-      (entry.period as any).text = formatStartEndDate(start, end);
-    }
+  const modifiedCareTeamParticipants: ModifiedCareTeamParticipant[] =
```

Comment:
not really for this PR, but this feels like a strange pattern. I wonder if it would be better to modify the `ColumnInfoInput` expectations, so you could provide an `evaluateFn: (entry, mapping) => string` which defaults to `evaluateValue`, but you could override 🤔